packages/core/src/physics/PhysicsScene.ts

```diff
@@ -374,10 +378,161 @@ export class PhysicsScene {
         return false;
       }
     } else {
-      return this._nativePhysicsScene.raycast(ray, distance, onRaycast);
+      return this._nativePhysicsScene.raycast(ray, distance, preFilter);
     }
   }
 
+  /**
+   * Casts a box through the Scene and returns true if there is any hit.
+   * @param center - The center of the box
+   * @param halfExtents - Half the size of the box in each dimension
+   * @param direction - The direction to sweep along
+   * @param orientation - The rotation of the box
+   * @param distance - The max distance to sweep, default is Number.MAX_VALUE
+   * @param layerMask - Layer mask that is used to selectively ignore Colliders when sweeping, default is Layer.Everything
+   * @returns Returns True if the box intersects with any collider, otherwise false
+   */
+  boxCast(
+    center: Vector3,
+    orientation: Quaternion,
```

Comment:
- Parameter order and annotations do not match
- Why is orientation not an optional parameter?